### DISCUSSION TITLE
2024-03-22-data-cache-%E7%9A%84%E5%A6%99%E7%94%A8/

### DISCUSSION THREAD

**Discussion by giscus** [williamlfang/williamlfang.github.io#2] [Announcements]
# 2024-03-22-data-cache-%E7%9A%84%E5%A6%99%E7%94%A8/

data cache 在用户调用资源开辟一块缓存区域，保存了中间的计算结果，可以避免重复计算。

https://williamlfang.github.io/2024-03-22-data-cache-%E7%9A%84%E5%A6%99%E7%94%A8/

**Comment by williamlfang** [comment]
cache 是一门极为优雅的手艺
